baseplate/_compat.py

```diff
@@ -14,16 +14,19 @@
     import queue
     from io import BytesIO
     range = range
+    string_types = str
```

Comment:
They could be, they aren't in `six`

https://github.com/benjaminp/six/blob/master/six.py#L41